src/tictactoe_clojure/rules.clj

```diff
@@ -9,12 +9,24 @@
       true 
       false))
 
+(defn switch-symbol
+  [previous-turn]
+    (if (= previous-turn "X")
```

Comment:
You've got `"X"` and `"O"` strings everywhere. Can you think of a way to eliminate these magic strings?